pyproject.toml

```diff
@@ -8,17 +8,25 @@ authors = ["Joel Wee <joel.wee@outlook.com>"]
 python = "^3.8"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
-requests_mock = "^1.8.0"
 gunicorn = "^20.0.4"
-pyyaml = "^5.3.1"
+pymongo = "^3.11.3"
+dnspython = "^2.1.0"
```

Comment:
Why is this necessary?

Reply:
It complains about this dependency if I don't include it